### ISSUE TITLE
Crash when building RBE example

### ISSUE THREAD

**Issue by jayconrod** [bazelbuild/bazel#11166]
### Description of the problem / feature request:

Bazel crashes with the error described below.

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

* Set up RBE, following the RBE Quickstart documentation (unfortunately only available to alpha customers, so I can't link it).
* Check out github.com/bazelbuild/bazel-toolchains at `810ac3490df9113cfaa50a4ee3d204a29c81a24c` (`master` as of 2020-04-20).
* `bazel --bazelrc=bazelrc/latest.bazelrc test //examples/remotebuildexecution/hello_world/cc:say_hello_test --config=remote --remote_instance_name=projects/bazel-go-remote/instances/default_instance`

This crashes with the error below.

<details><pre>
Starting local Bazel server and connecting to it...
WARNING: option '--host_javabase' was expanded to from both option '--config=remote' (source command line options) and option '--config=remote' (source command line options)
WARNING: option '--javabase' was expanded to from both option '--config=remote' (source command line options) and option '--config=remote' (source command line options)
WARNING: option '--crosstool_top' was expanded to from both option '--config=remote' (source command line options) and option '--config=remote' (source command line options)
WARNING: option '--host_platform' was expanded to from both option '--config=remote' (source command line options) and option '--config=remote' (source command line options)
WARNING: option '--platforms' was expanded to from both option '--config=remote' (source command line options) and option '--config=remote' (source command line options)
INFO: Invocation ID: d687c0eb-ed43-454f-a929-6d2fc1d458ca
DEBUG: Rule 'io_bazel_rules_docker' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1585331411 -0700"
DEBUG: Call stack for the definition of repository 'io_bazel_rules_docker' which is a git_repository (rule definition at /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_tools/tools/build_defs/repo/git.bzl:195:18):
 - <builtin>
 - /usr/local/google/home/jayconrod/go/src/github.com/bazelbuild/bazel-toolchains/repositories/repositories.bzl:35:9
 - /usr/local/google/home/jayconrod/go/src/github.com/bazelbuild/bazel-toolchains/WORKSPACE:22:1
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconfig_autogen_ubuntu1604 not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_ubuntu1604_trigger_config_gen_test not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:113:13: rbe_ubuntu1604_test not using checked in configs; registry was set to 'gcr.io' and toolchain_config_suite_spec is configured for 'marketplace.gcr.io'
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:113:13: rbe_ubuntu1804_test not using checked in configs; registry was set to 'gcr.io' and toolchain_config_suite_spec is configured for 'marketplace.gcr.io'
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_generate not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_generate_no_java not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_generate_no_cc not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:226:13: rbe_autoconf_config_repos not using checked in configs; Bazel version 3.0.0 was picked/selected with '["11.0.0"]' compatible configs but none match the 'env = None', 'config_repos = ["local_config_sh"]',and/or 'create_cc_configs = True' passed as attrs
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:226:13: rbe_autoconf_config_repos_no_cc_config not using checked in configs; Bazel version 3.0.0 was picked/selected with '["11.0.0"]' compatible configs but none match the 'env = None', 'config_repos = ["local_config_sh"]',and/or 'create_cc_configs = False' passed as attrs
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:103:9: rbe_autoconf_detect_java_home not using checked in configs as detect_java_home was set to True 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:185:13: rbe_autoconf_old_container not using checked in configs; config struct(config_repos = [], create_cc_configs = True, create_java_configs = True, env = {"ABI_LIBC_VERSION": "glibc_2.19", "ABI_VERSION": "clang", "BAZEL_COMPILER": "clang", "BAZEL_HOST_SYSTEM": "i686-unknown-linux-gnu", "BAZEL_TARGET_CPU": "k8", "BAZEL_TARGET_LIBC": "glibc_2.19", "BAZEL_TARGET_SYSTEM": "x86_64-unknown-linux-gnu", "CC": "clang", "CC_TOOLCHAIN_NAME": "linux_gnu_x86"}, java_home = "/usr/lib/jvm/java-8-openjdk-amd64", name = "8.0.0") was picked/selected, Bazel version 3.0.0 was picked/selected but no checked in config was found in ["11.0.0"]
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:113:13: rbe_autoconf_custom_container not using checked in configs; registry was set to 'l.gcr.io' and toolchain_config_suite_spec is configured for 'marketplace.gcr.io'
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:226:13: rbe_autoconf_custom_env not using checked in configs; Bazel version 3.0.0 was picked/selected with '["11.0.0"]' compatible configs but none match the 'env = {"ABI_LIBC_VERSION": "test_abi_libc_version_test", "ABI_VERSION": "test_abi_version_test", "BAZEL_COMPILER": "clang++", "BAZEL_HOST_SYSTEM": "test_bazel_host_system_test", "BAZEL_TARGET_CPU": "test_bazel_target_cpu_test", "BAZEL_TARGET_LIBC": "test_bazel_target_libc_test", "BAZEL_TARGET_SYSTEM": "test_bazel_target_system_test", "CC": "clang++", "CC_TOOLCHAIN_NAME": "test_cc_toolchain_name_test"}', 'config_repos = None',and/or 'create_cc_configs = True' passed as attrs
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:118:13: rbe_autoconf_resovle_tag not using checked in configs; repository was set to 'google/bazel' and toolchain_config_suite_spec is configured for 'google/rbe-ubuntu16-04'
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_output_base not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_output_base_no_java not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_output_base_no_cc not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_config_repos_output_base not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:100:9: rbe_autoconf_output_base_config_dir not using checked in configs as user set attr to 'False' 
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:113:13: rbe_autoconf_generate_no_docker_pull not using checked in configs; registry was set to 'gcr.io' and toolchain_config_suite_spec is configured for 'marketplace.gcr.io'
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:9: rbe_autoconf_custom_toolchain_config_suite_spec_export not using checked in configs; Bazel version 3.0.0 was picked/selected but no checked in config was found in map {"0.24.0": ["testConfigSpecName1"], "0.25.0": ["testConfigSpecName2"], "0.26.0": ["testConfigSpecName2", "testConfigSpecName1"]}
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:9: rbe_autoconf_custom_toolchain_config_suite_spec_blank_versions not using checked in configs; Bazel version 3.0.0 was picked/selected but no checked in config was found in map {}
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:125:9: rbe_autoconf_gcb_test not using checked in configs; Bazel version 3.0.0 was picked/selected but no checked in config was found in map {}
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/checked_in.bzl:185:13: buildkite_config not using checked in configs; config struct(config_repos = [], create_cc_configs = True, create_java_configs = True, env = {"ABI_LIBC_VERSION": "glibc_2.19", "ABI_VERSION": "clang", "BAZEL_COMPILER": "clang", "BAZEL_HOST_SYSTEM": "i686-unknown-linux-gnu", "BAZEL_TARGET_CPU": "k8", "BAZEL_TARGET_LIBC": "glibc_2.19", "BAZEL_TARGET_SYSTEM": "x86_64-unknown-linux-gnu", "CC": "clang", "CC_TOOLCHAIN_NAME": "linux_gnu_x86"}, java_home = "/usr/lib/jvm/java-8-openjdk-amd64", name = "10.0.0") was picked/selected, Bazel version 3.0.0 was picked/selected but no checked in config was found in ["11.0.0"]
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo.bzl:477:5: Bazel 3.0.0 is used in rbe_default.
INFO: Call stack for the definition of repository 'buildkite_config' which is a _rbe_autoconfig (rule definition at /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo.bzl:621:19):
 - <builtin>
 - /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo.bzl:1145:5
 - /usr/local/google/home/jayconrod/go/src/github.com/bazelbuild/bazel-toolchains/WORKSPACE:607:1
DEBUG: /usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo.bzl:477:5: Bazel 3.0.0 is used in buildkite_config.
ERROR: An error occurred during the fetch of repository 'buildkite_config':
   Traceback (most recent call last):
	File "/usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo.bzl", line 505
		validate_host(ctx)
	File "/usr/local/google/home/jayconrod/.cache/bazel/_bazel_jayconrod/30da858275e1fe9fb487b2332ed9ae5a/external/bazel_toolchains/rules/rbe_repo/util.bzl", line 152, in validate_host
		fail(<1 more arguments>)
Cannot run rbe_autoconfig as 'docker' was not found on the path and environment variable DOCKER_PATH was not set. rbe_autoconfig attempts to pull a docker container if a toolchain config was not found for the version of Bazel (selected via attr or implicitly identified). If you do not want rbe_autoconfig to ever attempt to pull a docker container, please use attr 'use_checked_in_confs = "Force"'.
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException: //examples/remotebuildexecution/hello_world/cc:say_hello_test BuildConfigurationValue.Key[a4058222645d06054277f3a046f0895d7fb4fe37852d098866adc0a0826496f4] false
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.devtools.build.skyframe.DelegatingWalkableGraph.getDirectDeps(DelegatingWalkableGraph.java:121)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.assertSaneAnalysisError(SkyframeBuildView.java:773)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.processErrors(SkyframeBuildView.java:616)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.configureTargets(SkyframeBuildView.java:454)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:404)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:213)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:145)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:290)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:131)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:109)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:564)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:208)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:603)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:659)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)

INFO: Elapsed time: 8.318s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (6 packages loaded, 10 targets con\
figured)
Internal error thrown during build. Printing stack trace: java.lang.NullPointerException: //examples/remotebuildexecution/hello_world/cc:say_hello_test BuildConfigurationValue.Key[a4058222645d06054277f3a046f0895d7fb4fe37852d098866adc0a0826496f4] false
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.devtools.build.skyframe.DelegatingWalkableGraph.getDirectDeps(DelegatingWalkableGraph.java:121)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.assertSaneAnalysisError(SkyframeBuildView.java:773)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.processErrors(SkyframeBuildView.java:616)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.configureTargets(SkyframeBuildView.java:454)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:404)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:213)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:145)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:290)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:131)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:109)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:564)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:208)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:603)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:659)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException: //examples/remotebuildexecution/hello_world/cc:say_hello_test BuildConfigurationValue.Key[a4058222645d06054277f3a046f0895d7fb4fe37852d098866adc0a0826496f4] false
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:895)
	at com.google.devtools.build.skyframe.DelegatingWalkableGraph.getDirectDeps(DelegatingWalkableGraph.java:121)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.assertSaneAnalysisError(SkyframeBuildView.java:773)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.processErrors(SkyframeBuildView.java:616)
	at com.google.devtools.build.lib.skyframe.SkyframeBuildView.configureTargets(SkyframeBuildView.java:454)
	at com.google.devtools.build.lib.analysis.BuildView.update(BuildView.java:404)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.runAnalysisPhase(AnalysisPhaseRunner.java:213)
	at com.google.devtools.build.lib.buildtool.AnalysisPhaseRunner.execute(AnalysisPhaseRunner.java:124)
	at com.google.devtools.build.lib.buildtool.BuildTool.buildTargets(BuildTool.java:145)
	at com.google.devtools.build.lib.buildtool.BuildTool.processRequest(BuildTool.java:290)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.doTest(TestCommand.java:131)
	at com.google.devtools.build.lib.runtime.commands.TestCommand.exec(TestCommand.java:109)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.execExclusively(BlazeCommandDispatcher.java:564)
	at com.google.devtools.build.lib.runtime.BlazeCommandDispatcher.exec(BlazeCommandDispatcher.java:208)
	at com.google.devtools.build.lib.server.GrpcServerImpl.executeCommand(GrpcServerImpl.java:603)
	at com.google.devtools.build.lib.server.GrpcServerImpl.lambda$run$2(GrpcServerImpl.java:659)
	at io.grpc.Context$1.run(Context.java:595)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown FAILED: Build did NOT complete successfully (6 packages loaded, 10 targets con\
figured)
</pre></details>

It looks like a repository rule is failing because `docker` is not installed, and something asserts that it doesn't fail.
### What operating system are you running Bazel on?

> Replace this line with your answer.

### What's the output of `bazel info release`?

release 3.0.0

**Comment by jin** [comment]
cc @EricBurnett

**Comment by ulfjack** [comment]
It's a bazel bug, but I haven't been able to reproduce. The crash happens in code that's supposed to sanity check the result, and by crashing it causes some information to be lost that would help debug.

My hypothesis is that the node throws RepositoryFetchException, which the sanity check doesn't expect in SkyframeBuildView.isSaneAnalysisError. I can't reproduce, so apparently, something is wrapping the RepositoryFetchException. I suspect that you may have additional options set. Do you have a .bazelrc in your home directory? If so, what does it contain?

**Comment by jayconrod** [comment]
@ulfjack I do have a ~/.bazelrc file. Here it is below. Would this be overridden by the `--bazelrc` argument though? Most of the options should be the same in any case.

```
#build --disk_cache=/Users/jayconrod/.bazelcache
test --test_output=errors

# RBE configuration
# https://cloud.google.com/remote-build-execution/docs/quickstart?hl=en
# Based on github.com/bazelbuild/bazel-toolchains/bazelrc/bazel-2.0.0.bazelrc
build:remote --jobs=50
build:remote --host_javabase=@buildkite_config//java:jdk
build:remote --javabase=@buildkite_config//java:jdk
build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
build:remote --crosstool_top=@buildkite_config//cc:toolchain
build:remote --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
build:remote --extra_toolchains=@buildkite_config//config:cc-toolchain
build:remote --extra_execution_platforms=@buildkite_config//config:platform
build:remote --host_platform=@buildkite_config//config:platform
build:remote --platforms=@buildkite_config//config:platform
build:remote --define=EXECUTOR=remote
build:remote --remote_executor=grpcs://remotebuildexecution.googleapis.com
build:remote --incompatible_strict_action_env=true
build:remote --remote_timeout=3600
build:remote --google_default_credentials=true

build:remote --remote_instance_name=projects/bazel-go-remote/instances/default_instance
```

**Comment by jayconrod** [comment]
@EricBurnett Thanks for the workaround. I was able to get this to work in rules_go, so I'm not blocked or anything. Just wanted to report the crash, since that's still a problem, even after changes in `buildkite_config` and `bazel_toolchains`.

**Comment by janakdr** [comment]
Closed by de6808488a5b97fd154fdb5dcdf28f030e60dbb8
